src/components/sections/items/ToggleSectionItem.tsx

```diff
@@ -67,37 +66,44 @@ export function ToggleSectionItem({
       onAccessibilityAction={() => onChange(!checked)}
       {...accessibility}
     >
-      <View style={sectionStyle.spaceBetween}>
-        {leftIcon && <ThemeIcon svg={leftIcon} style={styles.leftIcon} />}
-        <View style={styles.textContainer}>
-          <ThemeText type={textType}>{text}</ThemeText>
+      <View style={{flexDirection: 'row'}}>
+        <View style={styles.leftImageContainer}>{LeftImage && <LeftImage />}</View>
```

Comment:
Could the image be top-aligned?

Reply:
Currently no, but if we want to implement top-align I think we can use a prop for that instead of passing styles from the parent?